### PR TITLE
CaaS CRD driver

### PR DESCRIPTION
**Pull request by mkjpryor**
Supersedes #31

### CODE REVIEW

File: chart/files/api/settings/08-awx.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{{- if and .Values.tags.clusters (eq .Values.clusterEngine.type "awx") -}}
+AZIMUTH:
+  AWX:
+    ENABLED: true
```
Comment (JohnGarbutt) [inline chart/files/api/settings/08-awx.yaml:4]: Do we want the default to be false now?

**Review by JohnGarbutt — COMMENTED** [review]
OK, I think this means you can keep AWX and the CRD running still? Which is what we need I think.

File: chart/files/api/settings/08-awx.yaml (hunk 1 above)
Comment (mkjpryor) [reply chart/files/api/settings/08-awx.yaml:4]: It is behind a guard (see line 1). The default if not given has always been false - https://github.com/stackhpc/azimuth/blob/master/api/azimuth/settings.py#L23.

Comment (mkjpryor) [reply chart/files/api/settings/08-awx.yaml:4]: The default `.Values.clusterEngine.type` is now `crd` BTW

Comment (JohnGarbutt) [reply chart/files/api/settings/08-awx.yaml:4]: OK, I see, this is only included if you select awx as the clusterEngine?

Comment (mkjpryor) [reply chart/files/api/settings/08-awx.yaml:4]: Exactly

File: chart/files/api/settings/08-awx.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+{{- if and .Values.tags.clusters (eq .Values.clusterEngine.type "awx") -}}
```
Comment (JohnGarbutt) [inline chart/files/api/settings/08-awx.yaml:1]: oh... I missed this bit 🤣

**Review by JohnGarbutt — APPROVED** [review]

**Pull request merged**